system/Commands/Utilities/Routes.php

```diff
@@ -108,13 +117,23 @@ public function run(array $params)
                 $collection->getDefaultController(),
                 $collection->getDefaultMethod()
             );
-            $tbody = [...$tbody, ...$autoRouteCollector->get()];
+            $autoRoutes = $autoRouteCollector->get();
+
+            foreach ($autoRoutes as &$routes) {
+                $filters  = $filterCollector->get('get', $uriGenerator->get($routes[1]));
+                $routes[] = implode(' ', array_map('class_basename', $filters['before']));
+                $routes[] = implode(' ', array_map('class_basename', $filters['after']));
+            }
+
+            $tbody = [...$tbody, ...$autoRoutes];
         }
 
         $thead = [
             'Method',
             'Route',
             'Handler',
+            'Filters:before',
+            'Filters:after',
```

Comment:
Can we make these `Before Filters` and `After Filters`? Or those would be vague?

Reply:
I'm not sure which is better.

Reply:
Changed.